### DISCUSSION TITLE
Easter eggs

### DISCUSSION THREAD

**Discussion by jeffalo** [ScratchAddons/ScratchAddons#169] [Ideas]
**Is your feature request related to a problem? Please describe.**

i was looking and i noticed there was not enough funny. 

**Describe the solution you'd like**

adding the funny

**Describe alternatives you've considered**

going down the route of [microsoft ](https://docs.microsoft.com/en-us/archive/blogs/larryosterman/why-no-easter-eggs)and other silly companies

**Additional context**

so i dont care what easter eggs but this is quite important for reasons therefore it needs to exist.

ok so some ideas include:
the konami code
appel
appel
i cant think of any more ideas but you get the idea

**Comment by retronbv** [comment]
yes plz

**Comment by Hans5958** [comment]
let's start putting witty comments on the `.js`

**Comment by Explosion-Scratch** [comment]
lol yeah

**Comment by Hans5958** [comment]
ot: 17 issues and 2 prs on a single day and counting. wow, this is nuts

**Comment by jeffalo** [comment]
> ot: 17 issues and 2 prs on a single day and counting. wow, this is nuts

yeah its crazy! i planned to make a huge PR with a ton of userstyles today but imagine the git conflicts!

**Comment by jeffalo** [comment]
> let's start putting witty comments on the `.js`

https://github.com/ScratchAddons/ScratchAddons/blob/master/addons/image-uploader/userscript.js#L103

**Comment by WorldLanguages** [comment]
![image](https://user-images.githubusercontent.com/17484114/94495112-4accfd80-01c7-11eb-849e-d48c737be8cd.png)
Can we add something like this for the settings page, if no results were found for the search term? :P

**Comment by Explosion-Scratch** [comment]
I really love the gif (on the 404 page of my website) here:
https://explosion-scratch.github.io

On Monday, September 28, 2020, World_Languages <notifications@github.com>
wrote:
> <
https://ci3.googleusercontent.com/proxy/QibPpPf95Jfm4V64fb_HCHOFF3JlpLpSPZIFShhwx4MqIEDLkv8wvPpLM444W0pxfSzliQ980u-RlWr8bDzFOh8MqvFLNHllduWmdk3-pxiEFhRRoAXhOa6x0SEO-haP_oRfX4qXrrzb6VnN9NiEI_ucGgEPlwY=s0-d-e1-ft#https://user-images.githubusercontent.com/17484114/94495112-4accfd80-01c7-11eb-849e-d48c737be8cd.png
>
> Can we add something like this for the settings page, if no results were
found for the search term? :P
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or unsubscribe.<
https://ci5.googleusercontent.com/proxy/GIDNiKGWC997yW7ixfufJi1QYgctuvnOA46ILdnJDaNg1hqr_NrymNhHm8XhsT5paSvV74B3OPVzZCm3vQHToaG_tdABvHTvcSfNzQsHMWq4rhXFSOyr8Z41JmEwBARVsq7JljokEklMSY-ZVR1n-9iCblO6YiO5U66mEmlkEsmL7hh3O4sCCi5cdvav9-DcoQS_uYTFa8hDx5GeYngMVeZqCd4StiG9tS4McsyuHQ=s0-d-e1-ft#https://github.com/notifications/beacon/AOT2P3RHUKVQ5AKDILAA5NTSIEKJNA5CNFSM4QTEOAB2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOFG7DNQA.gif>

**Comment by jeffalo** [comment]
please can add appel? this is the very important

**Comment by WorldLanguages** [comment]
Moving to #508
